kie-internal/pom.xml

```diff
@@ -166,12 +167,20 @@
     <dependency>
       <groupId>com.sun.xml.bind</groupId>
       <artifactId>jaxb-xjc</artifactId>
-      <scope>provided</scope>
+       <scope>compile</scope>
+       <optional>true</optional>
```

Comment:
@MarianMacik how about these dependencies in context of JDK 11 work you're doing?

Reply:
Hi, I am doing things mostly regarding APIs, so implementations themselves should be good.

Reply:
@elguardian why do we need xjc?

Reply:
@mswiderski testing. they fail during unmarshalling if that dependency is not there.

Reply:
but does it have to be compile scope?

Reply:
fixed to provided